workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+ 
+Firstly, let’s go to style.css.
+
+Now add the following code:
+
+```css
+*{
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  outline: none;
+}
+
+body{
+  font-family: "Open Sans";
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  overflow: hidden;
+  background: rgb(60, 60, 242);
+  background: linear-gradient(90deg, rgba(60, 60, 242, 1) 0%, rgba(98, 245, 230, 1) 52%, rgba(60, 60, 242, 1) 100%);
+  background-size: cover;
+}
+```
+      
+### Explanation: 
+3 main steps in writing CSS:
+* Select the element/class/id on which you want to apply your styles.
+* Open curly brackets {}.
+* Start adding your styles inside the curly brackets!
+
+In the first line, we have given the size of the box (triangles of our wheel). Then in the second- & third-line padding & margin between them is marked as zero respectively as we don’t require space between the triangles and in fourth line outline is also kept none as we don’t require border when we click on the spin button.
+
+![spin button with border](https://cloud-migxi228s.vercel.app/0spin1.png)
+
+^^^ If we don’t give `outline: none;` then our button will look like this after clicking.
+
+![spin button without border](https://cloud-migxi228s.vercel.app/1spin2.png)
+
+^^^ Whereas if we give `outline: none;` then our button will look like this (without any border) after clicking.
+
+Now in the next block, we have `font-family: Open Sans;` this syntax is used to give the name/type of font we want to have in our block. Here we have used it in selector as body therefore it will be applied as a default font on our whole webpage.
+
+We changed the default display `display: block` to `display: flex`.
+
+A very good reason for this is because flex enables you to align your items perfectly to the center! And that's what we have done in the next two lines. The align-items property aligns all your children elements to center, but horizontally. The justify-content property aligns all your children elements to center, but vertically.
+
+Next, we have the min-height property which sets a minimum height of 100vh (viewport height) to your <body>. The min-height property works together with the align-items property and helps the children to align properly at the center.
+
+Learn more about [viewport-heights](https://www.sitepoint.com/css-viewport-units-quick-start/).
+In the background section, you can use any color’s hex code, but in this workshop, I have used gradient which you may get from the [CSS gradient](https://cssgradient.io/). Go on that website and make your gradient then copy the code given below and paste it in the background section. By this, your custom background will be applied.
```

Comment:
- What is "the background section"? I assume this means the `background` selector, but I think it should be clearer. 
- "I have used gradient" awkward phrasing
- "gradient which you may get from" –> "gradient, which you can get from"
- instructions for making a gradient should be a little clearer, not in a run-on sentence, and should probably be in a new paragraph